.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```

Comment:
:red_circle: Please kindly remove the .vscode folder from your project directory. It should not be there because it has nothing to do with your project. :pray: You can delete it from GitHub and make sure you include the folder `.vscode`  in your .gitignore file.